src/mol-util/nodejs-browser-io.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2018-2020 mol* contributors, licensed under MIT, See LICENSE file for more info.
+ *
+ * @author Adam Midlik <midlik@gmail.com>
+ *
+ * Implements some browser-only global variables for Node.js environment.
+ * These workarounds will also work in browsers as usual.
+ */
+
+
+/** Determines whether the current code is running in Node.js or in a browser */
+export const RUNNING_IN_NODEJS = typeof document === 'undefined';
```

Comment:
there other runtime times... this checks if it is running in a browser, to check for node do

```
const isNode =
  typeof process !== "undefined" &&
  process.versions != null &&
  process.versions.node != null;
```